content/source/docs/cloud/users-teams-organizations/single-sign-on/custom-saml.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: "cloud"
+page_title: "SAML - Single Sign-on - Terraform Cloud and Terraform Enterprise"
+---
+
+-> **Note:** Single sign-on is a paid feature, available as part of the **Business** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
+
+# Single Sign-on: SAML
+
+The SAML SSO integration currently supports the following features of SAML 2.0:
+
+- Service Provider (SP)-initiated SSO
+- Identity Provider (IdP)-initiaited SSO
```

Comment:
Nit. This is odd (from a US perspective). Typically we'd write Service Provider (SP) initiated...

Reply:
Ah, it might be that I copied it from Okta and then added the full word beforehand and parenthesised the acronym. @nfagerlund preference?

> SP-initiated SSO

Is what Okta has.